client/my-sites/plugins-wpcom/plugin.jsx

```diff
@@ -18,11 +28,29 @@ export const StandardPlugin = React.createClass( {
 			descriptionLink
 		} = this.props;
 
+		const { isUnderMouse } = this.state;
+
+		const isExternalLink = hasHttpProtocol( descriptionLink );
```

Comment:
@gwwar we deliberated on this point and I think it's best to keep the logic contained inside this component. it's only effect is visual and I don't think there's any need to couple that to the "outside world" I like being able to only care about adding the URL in the plugin data structure instead of having to think about how to represent that data.